pegasus/helpers.rb

```diff
@@ -1,6 +1,8 @@
 require 'cdo/aws/s3'
 require 'rack/csrf'
 require_relative '../shared/middleware/helpers/storage_id'
+require_relative '../shared/middleware/helpers/core.rb'
+require_relative '../shared/middleware/helpers/auth_helpers.rb'
```

Comment:
things required here appear to be available in pegasus unit tests, which is not true for things required in config.ru